Swiftx/Either.swift

```diff
@@ -51,7 +51,7 @@ public enum Either<L, R> {
 
 	/// Case analysis for the Either type. If the value is Left(a), apply the first function to a;
 	/// if it is Right(b), apply the second function to b.
-	public func either<A>(#onLeft : L -> A, onRight : R -> A) -> A {
```

Comment:
It's totally my fault if that's the case.  I'm trying to push for labels in the next release after this and accidentally included some in SwiftX.